### PR TITLE
Feature/fix invoice line hook + pt_br Translate

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Patch looks good, but I cannot say about translation.

**Comment by mileo** [comment]
This is the same translation of https://github.com/OCA/commission/pull/5

Sorry about the translation, we have a problem with a mirror this week and lost some branchs. 

@renatonlima Vc poderia dar uma revisada? 

Regards

File: sale_commission/i18n/pt_BR.po [hunk 1]
```diff
@@ -80,7 +79,7 @@ msgstr "Resumo da fatura"
 #. module: sale_commission
 #: sql_constraint:sale.order:0
 msgid "Order Reference must be unique !"
-msgstr "Referência à ordem deve ser único!"
+msgstr "Referência à ordem deve ser única!"
```
Comment (danimaribeiro) [inline sale_commission/i18n/pt_BR.po:82]: "Referência do pedido deve ser única!" 
This one looks better.

**Comment by danimaribeiro** [comment]
I added one note to one translation.
The rest looks good, feel free to merge.

File: sale_commission/stock_picking.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def _invoice_line_hook(self, cr, uid, move_line, invoice_line_id, context=None):
         if context is None:
             context = {}
         agent_pool = self.pool.get('invoice.line.agent')
-        super(stock_picking, self)._invoice_line_hook(cr, uid, move_line, invoice_line_id, context=context)
+        super(stock_picking, self)._invoice_line_hook(cr, uid, move_line, invoice_line_id)
```
Comment (pedrobaeza) [inline sale_commission/stock_picking.py:51]: You shouldn't change this

Comment (foutoucour) [inline sale_commission/stock_picking.py:51]: Hey,

_invoice_line_hook doesn't take context as argument anymore in fact. Without this patch, the tests of commission fails

**Comment by foutoucour** [comment]
:+1: 
The fails come from pep8 issues from the previous version.

File: sale_commission/stock_picking.py (hunk 2 above)
Comment (pedrobaeza) [inline sale_commission/stock_picking.py:51]: OK, thanks for explaining

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**